README.md

```diff
@@ -145,74 +152,27 @@ sumAgg := view.SumAggregation{}
 meanAgg := view.MeanAggregation{}
 ```
 
-### Creating, registering and unregistering a view
-
-Create and register a view:
+Here we create a view with the DistributionAggregation over our Measure.
+Setting tags to nil (as in this example) means that Measurements with all tags
+will be aggregated together (no grouping by tag):
 
 [embedmd]:# (stats.go view)
 ```go
-v, err := view.New(
-	"my.org/video_size_distribution",
-	"distribution of processed video size over time",
-	nil,
-	videoSize,
-	distAgg,
-)
-if err != nil {
-	log.Fatalf("cannot create view: %v", err)
-}
-if err := view.Register(v); err != nil {
-	log.Fatal(err)
+v := &view.View{
+	Name:        "my.org/video_size_distribution",
+	Description: "distribution of processed video size over time",
+	MeasureName: videoSize.Name(),
```

Comment:
I'd take the Measure here even though I will internally use the measure name. API should work with the high-level APIs if possible.